gulp/common-tasks/scss.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (gulp, plugins, config, destFolder = 'assets', addSrc)
         browsers: plugins.supportedBrowser,
         cascade: false,
       }))
-      .pipe(sourcemaps.write())
+      .pipe(sourcemaps.write('.'))
```

Comment:
what does this do?

Reply:
write maps file for debugging in the same folder for local development. Earlier it was outputting in the root, so Chrome could not detect it.